.env.example

```
@@ -1,95 +1,169 @@
+LOG_FILE=logs/crawlerr.log
+LOG_TO_FILE=true
+PID_FILE=logs/crawlerr.pid
 
-# Qdrant Vector Database
+# =============================================================================
+# SERVICE ENDPOINTS
+# =============================================================================
+# Qdrant Vector Database - Performance Optimized
 QDRANT_URL=http://localhost:7000
-# QDRANT_API_KEY=your_api_key_here
+QDRANT_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The QDRANT_API_KEY key should go before the QDRANT_URL key

(UnorderedKey)